ingest/config/config.yaml

```diff
@@ -70,7 +70,5 @@ transform:
     'date_submitted',
     'sra_accession',
     'abbr_authors',
-    'reverse',
     'authors',
-    'institution'
```

Comment:
_non-blocking_

FWIW, `institution` can be populated by adding `submitter-affiliation=institution` to the `field_map` config.

Reply:
Thanks! Looking around the code configs, I decided to add a functionally populated `institution` column in https://github.com/nextstrain/rsv/pull/70/commits/644faa9e3e910f4e4a63c7de04952674a081a2d1